### PR TITLE
Optimize build size

### PR DESCRIPTION
**Pull request by shammowla**
1) Updated dependencies
2) Rollup visualizer provides a visual breakdown of the minified code
3) Refactored babel config

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have made any necessary test changes and all tests pass.
- [ ] I have run the Sandbox successfully.

### CODE REVIEW

**Comment by jfkhoury** [comment]
@shammowla Good work. 

2 Requests, please:

1. Can you add comments to the main changes in this PR explaining why we did them? (Comments in Github, not code)
2. Can you add details to the description such as library size pre and post changes

**Comment by carterworks** [comment]
For reference, 

The build size for `optimize-build`

```
 28K	dist/alloy.min.js.gz
 84K	dist/alloy.min.js
388K	dist/alloy.js
500K	dist
```

The build size for `main`

```
 28K	dist/alloy.min.js.gz
 88K	dist/alloy.min.js
420K	dist/alloy.js
536K	dist
```

Workflow: `npm ci && npm run clean && npm run build && gzip --keep dist/alloy.min.js > dist/alloy.min.js.gz && du -ah dist/`

File: package.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
   "devDependencies": {
     "@adobe/alloy": "^2.15.0",
     "@babel/cli": "^7.12.8",
-    "@babel/core": "^7.2.2",
+    "@babel/core": "7.18.6",
```
Comment (carterworks) [inline package.json:67]: Why lock this to a specific version?

Comment (shammowla) [reply package.json]: Good catch, I removed the lock.

**Comment by jfkhoury** [comment]
Thank you so much for the effort @shammowla! Since the outcome is only 2kb, I suggest we close this PR and switch out efforts into optimizing the extension.

**Pull request closed without merging**